src/components/search-bar/search-bar.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import './search-bar.css';
+
+/**
+ * @typedef SearchBar
```

Comment:
I think this is the only functional issue in this PR.

```suggestion
 * @typedef SearchBarProps
```